topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -437,7 +451,7 @@ The configuration is quite simple thanks to the many sensible defaults that are
 >    > > galaxy_layout: root-dir
 >    > > galaxy_root: /srv/galaxy
 >    > > galaxy_user: {name: galaxy, shell: /bin/bash}
->    > > galaxy_commit_id: release_19.05
+>    > > galaxy_commit_id: release_20.01
 >    > > galaxy_config_style: yaml
 >    > > galaxy_force_checkout: true
```

Comment:
```suggestion
>    > > galaxy_force_checkout: true
>    > > miniconda_version: "latest"
```

Reply:
So in this tutorial I'd tried to avoid things that would mutate over time in unxpected ways, then when people use the training archive, they can really reproduce it exactly. Hence the pinning of all the ansible roles. This feels like an unpinned ansible role, is it important to have a newer conda? Could we pin to a specific newer conda that we know works?

Reply:
Let's go with 4.6.14 which is the current default conda version